glue/clients/ds9norm.py

```diff
@@ -123,8 +123,8 @@ def __init__(self):
         self.stretch = 'linear'
         self.bias = 0.5
         self.contrast = 1.0
-        self.clip_lo = 5.
-        self.clip_hi = 95.
+        self.clip_lo = 0.
+        self.clip_hi = 100.
```

Comment:
Just to clarify, the contrast and bias would still get set by the mouse mode, but I'm suggesting to move the selection of the function and percentile to the options widget.